src/Firehose.Web/Authors/AlessandroCaliaro.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.ServiceModel.Syndication;
+using Firehose.Web.Infrastructure;
+
+namespace Firehose.Web.Authors
+{
+    public class AlessandroCaliaro : IAmACommunityMember
+    {
+        public string FirstName => "Alessandro";
+        public string LastName => "Caliaro";
+        public string ShortBioOrTagLine => "software developer for mobile devices";
```

Comment:
Maybe write something more exciting. Also unclear what you mean here. This implies that you develop mobile devices.